Gem/Transfer/ViewsModels/PriceImpactViewModel.swift

```diff
@@ -0,0 +1,91 @@
+// Copyright (c). Gem Wallet. All rights reserved.
+
+import Foundation
+import Primitives
+import Localization
+
+enum PriceImpactValue: Equatable {
+    case none
+    case low(String)
+    case medium(String)
+    case high(String)
+    case positive(String)
+}
+
+struct PriceImpactViewModel {
+    let fromAssetData: AssetData
+    let toAssetData: AssetData
+    
+    var priceImpact: String { Localized.Swap.priceImpact }
+    
+    private let valueFormatter = ValueFormatter(style: .medium)
+    private var percentFormatter: NumberFormatter {
+        let formatter = NumberFormatter()
+        formatter.locale = .current
+        formatter.numberStyle = .percent
+        formatter.maximumFractionDigits = 2
+        formatter.minimumFractionDigits = 2
+        return formatter
+    }
+    
+    // MARK: - Public methods
+    
+    func priceImpactValue(fromValue: String, toValue: String) -> PriceImpactValue {
```

Comment:
```suggestion
    func value(fromValue: String, toValue: String) -> PriceImpactValue {
```

model is already called `PriceImpactViewModel`